pkg/webui/console/store/actions/packet-broker.js

```diff
@@ -223,3 +223,50 @@ export const [
     failure: getHomeNetworkRoutingPoliciesFailure,
   },
 ] = createRequestActions(GET_HOME_NETWORK_ROUTING_POLICIES_BASE)
+
+export const GET_HOME_NETWORK_DEFAULT_GATEWAY_VISIBILITY_BASE =
+  'GET_HOME_NETWORK_DEFAULT_GATEWAY_VISIBILITY'
+export const [
+  {
+    request: GET_HOME_NETWORK_DEFAULT_GATEWAY_VISIBILITY,
+    success: GET_HOME_NETWORK_DEFAULT_GATEWAY_VISIBILITY_SUCCESS,
+    failure: GET_HOME_NETWORK_DEFAULT_GATEWAY_VISIBILITY_FAILURE,
+  },
+  {
+    request: getHomeNetworkDefaultGatewayVisibility,
+    success: getHomeNetworkDefaultGatewayVisibilitySuccess,
+    failure: getHomeNetworkDefaultGatewayVisibilityFailure,
+  },
+] = createRequestActions(GET_HOME_NETWORK_DEFAULT_GATEWAY_VISIBILITY_BASE)
+
+export const SET_HOME_NETWORK_DEFAULT_GATEWAY_VISIBILITY_BASE =
+  'SET_HOME_NETWORK_DEFAULT_GATEWAY_VISIBILITY'
+export const [
+  {
+    request: SET_HOME_NETWORK_DEFAULT_GATEWAY_VISIBILITY,
+    success: SET_HOME_NETWORK_DEFAULT_GATEWAY_VISIBILITY_SUCCESS,
+    failure: SET_HOME_NETWORK_DEFAULT_GATEWAY_VISIBILITY_FAILURE,
+  },
+  {
+    request: setHomeNetworkDefaultGatewayVisibility,
+    success: setHomeNetworkDefaultGatewayVisibilitySuccess,
+    failure: setHomeNetworkDefaultGatewayVisibilityFailure,
+  },
+] = createRequestActions(SET_HOME_NETWORK_DEFAULT_GATEWAY_VISIBILITY_BASE, visibility => ({
+  visibility,
+}))
+
+export const DELETE_HOME_NETWORK_DEFAULT_GATEWAY_VISIBILITY_BASE =
+  'DELETE_HOME_NETWORK_DEFAULT_GATEWAY_VISIBILITY'
+export const [
+  {
+    request: DELETE_HOME_NETWORK_DEFAULT_GATEWAY_VISIBILITY,
+    success: DELETE_HOME_NETWORK_DEFAULT_GATEWAY_VISIBILITY_SUCCESS,
+    failure: DELETE_HOME_NETWORK_DEFAULT_GATEWAY_VISIBILITY_FAILURE,
+  },
+  {
+    request: deleteHomeNetworkDefaultGatewayVisibility,
+    success: deleteHomeNetworkDefaultGatewayVisibilitySuccess,
+    failure: deleteHomeNetworkDefaultGatewayVisibilityFailure,
+  },
+] = createRequestActions(DELETE_HOME_NETWORK_DEFAULT_GATEWAY_VISIBILITY_BASE)
```

Comment:
I didn't delete this action, in case it is needed in the future, although I am not currently using it in the form.